Fix for DW-1041: Adding troubleshooting note on reader properties usa… (#97)
mulesoft/docs-dataweave: modules/ROOT/pages/dataweave-troubleshoot.adoc

`application/java`, and the content is handled as a `String` instead of a
JSON object.

=== Reader properties not working inside a Mule application

> When working with a transformation inside a Mule application, `input` directives declared inside a script do not work.

Reviewer: ```suggestion
In a Mule application, the `input` directive to a DataWeave script does not work.
```

> This produces a different behaviour when running the same script in a standalone DataWeave runtime like the one powering the playground.

Reviewer: ```suggestion
Unlike Mule runtime, a standalone DataWeave runtime, such as the runtime in the https://developer.mulesoft.com/learn/dataweave/[DataWeave Playground+], can process a valid MIME type set through the `input` directive in the same DataWeave script.
```

Reviewer: @martincousido We need to say what behavior it produces. Does it input the specified MIME type if it is supported by DataWeave? Pls review the suggested change.

Author: Basically what happens is that any reader property that you want to specify on your inputs directly in the script, won't work on a mule environment, you have to go and set it in the output of the Message Processor where the data comes from. It's a very generic recommendation because there are multiple cases where this can happen, for example settings a custom separator for a csv file.

Author: I re-read it and I think the suggestion is good

Author: Only one thing, I think we could remove one runtime.
```suggestion
Unlike Mule runtime, a standalone DataWeave runtime, such as the one in the https://developer.mulesoft.com/learn/dataweave/[DataWeave Playground+], can process a valid MIME type set through the `input` directive in the same DataWeave script.
```

> Reader properties need to be configured on the MIME type `outputMimeType` attribute of the source of the data to produce

Reviewer: ```suggestion
To input reader properties to a script in a Mule application, configure the `outputMimeType` attribute for the data source to produce
```
